lib/routes/scientificamerican/podcast.ts

```diff
@@ -0,0 +1,270 @@
+                    const authors: DataItem['author'] = articleData.authors.map((author) => ({
+                        name: author.name,
+                        url: author.url ? new URL(author.url, baseUrl).href : undefined,
+                        avatar: author.picture_file,
+                    }));
+                    const guid: string = `scientificamerican-${articleData.id}`;
+                    const updated: number | string = articleData.updated_at_date_time ?? pubDate;
+
+                    let processedItem: DataItem = {
+                        title,
+                        description,
+                        pubDate: pubDate ? timezone(parseDate(pubDate), +8) : undefined,
+                        category: categories,
+                        author: authors,
+                        doi: articleData.article_doi,
+                        guid,
+                        id: guid,
+                        content: {
+                            html: description,
+                            text: articleData.summary ?? description,
+                        },
+                        image,
+                        banner: image,
+                        updated: updated ? timezone(parseDate(updated), +8) : undefined,
+                        language,
+                    };
+
+                    const enclosureUrl: string | undefined = articleData.media_url;
+
+                    if (enclosureUrl) {
+                        const enclosureType: string = `audio/${enclosureUrl.replace(/\?.*$/, '').split(/\./).pop()}`;
+
+                        processedItem = {
+                            ...processedItem,
+                            enclosure_url: enclosureUrl,
+                            enclosure_type: enclosureType,
+                            enclosure_title: title,
+                            itunes_item_image: image,
+                        };
+                    }
+
+                    return {
+                        ...item,
+                        ...processedItem,
+                    };
+                });
+            })
+        )
+    ).filter((_): _ is DataItem => true);
+
+    return {
+        title: $('title').text(),
+        description: $('meta[name="description"]').attr('content'),
+        link: targetUrl,
+        item: items,
+        allowEmpty: true,
+        image: $('meta[property="og:image"]').attr('content'),
+        author: $('meta[property="og:site_name"]').attr('content'),
+        language,
+        feedLink: $('link[type="application/rss+xml"]').attr('href'),
+        itunes_author: $('meta[property="og:site_name"]').attr('content'),
+        itunes_category: 'Science',
+        id: $('meta[property="og:url"]').attr('content'),
+    };
+};
+
+export const route: Route = {
+    path: ['/podcasts/:id?', '/podcast/:id?'],
```

Comment:
```suggestion
    path: '/podcast/:id?',
```
Please specify one path only

Reply:
😊